Week7/week7 solution.py

```diff
@@ -53,34 +53,34 @@
 df = df.append(new_rows, ignore_index=True)
 df.reset_index(drop=True, inplace=True)

```

Comment:
what is use of inplace=true?

Reply:
The inplace=True parameter is a boolean parameter that is used to modify the original data frame in place.